axolotl-web/src/pages/DeviceList.vue

```diff
@@ -1,20 +1,14 @@
 <template>
   <div class="deviceList">
-    <!-- eslint-disable vue/no-use-v-if-with-v-for,vue/no-confusing-v-for-v-if -->
-    <div v-if="devices && devices.length>1">
-      <div
-        class="row device"
-        v-for="device in devices"
-        v-if="device.id != 1 && false"
-        v-bind:key="device.id"
-      >
-        <div class="col-10">
+    <div v-if="devices && devices.length > 1">
```

Comment:
Looks good. Only thing I thought about was this check: if the devices array only contains one element, we dont want to show it in the list?

As far as I read it, the logic only evaluates to true if devices.length has a value of two or larger.

Reply:
Compared to a `devices.length >= 1` check, which still shows the list also on only one element.

Reply:
The first device item is axolotl itself. If we allow to remove that one it's like de-registering. I did it already twice :). Maybe I add an comment why it's that way.